<?xml version="1.0"?>
<robot name="cube_stack_arm" xmlns:xacro="http://ros.org/wiki/xacro">

  <!-- Attach Arm to Ground -->
  <link name="world"></link>

  <xacro:include filename="$(find cube_stack_env)/urdf/turtlebot_arm/turtlebot_arm.xacro"/>
  <xacro:turtlebot_arm parent="world" color="White" gripper_color="Green" control="effort">
    <origin xyz="0 0 0"/>
  </xacro:turtlebot_arm>

  <xacro:include filename="$(find cube_stack_env)/urdf/sensors.xacro"/>
  <xacro:depth_camera camera_name="depth_camera" parent_link="arm_base_link" far_range="1.0">
    <box size="0.01 0.01 0.01"/>
    <origin xyz="0.6 0 0.1" rpy="0 0 3.14"/>
  </xacro:depth_camera>

</robot>